.gitignore

```diff
@@ -14,3 +14,5 @@
 /log/*.log
 /tmp
 /doc
+
+coverage
```

Comment:
VSCode setting to insert new line at the end of files:`"files.insertFinalNewline": true,`